run.sh

```diff
@@ -42,11 +45,16 @@ else
 fi
 
 if [ ! -e "$JTREG_HOME" ] ; then
-  ball=jtreg-7+1.tar.gz
+  ball=jtreg5.1-b01.tar.gz
```

Comment:
Why the downgrade? Bytecode comaptiblity?
If so,then the switch on java -version  2>&1 | grep 1.8.0  or jshell  with `System.out.print(Runtime.version().major()`  shoudlbe added....

Reply:
Yes it is due to bytecode compatibility. Why not just use older jtreg, which runs everywhere? Also jdk8 does not have jshell.

Reply:
sure jdk 8 do not have jshell, thats why the  `if java -version 2>&1 | grep 1.8.0`....
Why not the older by default? because it wills top working soon.

But nvm. Just thinking laudly. Ok to merge for sure. TY!